comp/core/tagger/component.go

```diff
@@ -45,7 +45,6 @@ type Component interface {
 	LegacyTag(entity string, cardinality types.TagCardinality) ([]string, error)
 	Tag(entityID types.EntityID, cardinality types.TagCardinality) ([]string, error)
 	AccumulateTagsFor(entityID types.EntityID, cardinality types.TagCardinality, tb tagset.TagsAccumulator) error
-	Standard(entityID types.EntityID) ([]string, error)
```

Comment:
It's still used:
https://github.com/DataDog/datadog-agent/blob/debb814cb51c8106402d279f0bce656d62b2b434/pkg/logs/launchers/container/tailerfactory/defaults.go#L37

Reply:
Ohhh what a bummer 😢